docs/src/pages/guides/notifications.md

```diff
@@ -343,11 +343,11 @@ You can get a notification from Monika to your Whatsapp number without having a
 
 ## Desktop Notifications
 
-Monika uses [node-notifier](https://github.com/mikaelbr/node-notifier#readme) to support desktop notifications. Here are the prerequisites for enabling the desktop notifications:
+Monika supports desktop notifications. Here are the prerequisites for enabling the desktop notifications:
```

Comment:
this `supports` suppose to be plural or singular ya?

Reply:
`supports` uses `s` because of the subject (Monika)